core/src/test/java/org/mobilitydata/gtfsvalidator/notice/OtherTestValidationNotice.java

```diff
@@ -0,0 +1,35 @@
+ /*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import java.util.Map;
+
+public class OtherTestValidationNotice extends ValidationNotice {
```

Comment:
Is that class reused in other tests? If not, please make it a static private class in the test where it is used.

Reply:
It is not, will do. Thanks!